api/types/installers/installer.sh.tmpl

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
+
+on_azure() {
+  AZURE_STATUS=$(curl -o /dev/null -w "%{http_code}" -m5 -sS -H "Metadata:true" --noproxy "*" "http://169.254.169.254/metadata/versions")
```

Comment:
Could you run shellcheck here and fix whatever it complains about?

Reply:
No complaints from shellcheck.